### DISCUSSION TITLE
Ocean Life & Baloon Animation

### DISCUSSION THREAD

**Discussion by Racooder** [SebLague/Geographical-Adventures#89] [Ideas]
1. It would make the ocean a lot less empty if there were some ships or animals like dolphins, wales or rays.
2. Also I don't really like the animation when the hot air ballons disappear. Something like a short upward motion, shrinking to a small size and then a fast downward motion to the ground would maybe be better.

Additionally I find it really hard to assess the height of the baloons but I don't really can think of an solution.
I'm open for additions, corrections or solutions.

**Comment by evilkillerfiggin** [comment]
Yeah, I think the collider for the balloons should be a cylinder reaching all the way up and all the way down - so you just have to fly over the city at any height and you'll pick it up.

Accurately targeting the balloon in 3D space isn't really the point of the game.

**Reply by Racooder** [reply]
That's a really good point.
